src/sentry/snuba/discover.py

```diff
@@ -639,7 +639,10 @@ def create_result_key(result_row, fields, issues) -> str:
     values = []
     for field in fields:
         if field == "issue.id":
-            values.append(issues.get(result_row["issue.id"], "unknown"))
+            issue_id = issues.get(result_row["issue.id"], "unknown")
+            if issue_id is None:
```

Comment:
A rudimentary test for this would be amazing.

Reply:
added in b6705df